packages/marketplace/src/logic/Utils.ts

```diff
@@ -317,6 +318,109 @@ export const initColorShiftLoop = (schemes: SchemeIni) => {
   }, 60 * 1000);
 };
 
+export const getColorFromImage = async (image: HTMLImageElement, numColors : number) => {
+  const swatches = await Vibrant.from(image).maxColorCount(numColors).getPalette((err, palette) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+    return palette;
+  });
+  if (swatches.Vibrant) {
+    // remove the # from the hex
+    return swatches.Vibrant.hex.substring(1);
+
+  }
```

Comment:
Why are there do many random newlines lol
```suggestion
    return swatches.Vibrant.hex.substring(1);
  }
```